test/views/SummarySubscriptionsViewSpec.scala

```diff
@@ -18,6 +18,7 @@ package views
 
```

Comment:
application stop

Reply:
None of the rest of the view specs have application.stop()